portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,208 @@
+    background-image: url("Images/pinkd(1).jpg");
+    background-size: 100%;
+    height: 700px;
+    background-repeat:no-repeat;
+    background-position: center;
+
 
-#greeting-container {
-  margin-top: 20px;
 }
+  
+/***** About me *****/
+
+.about-box {
+    margin-top: 100px;
+
+}
+ 
+.about-box-text {
+    padding-right: 30px;
+    text-align: left;
+}
+ 
+.about-box-text h3 {
+    margin-top: 0;
+}
+ 
+.about-box-text .medium-paragraph {
+    margin-top: 20px;
+    margin-bottom: 20px;
+}
+ 
+.about-box-text a {
+    border-bottom: 1px dashed #9b59b6;
+}
+ 
+.about-box-text a:hover,
+.about-box-text a:focus {
+    border: 0;
+}
+ 
+.about-box-img img {
+    border-radius: 4px;
+}
+
+
+/*project section*/
+
+.portfolio-box {
+    padding-top: 30px;
+    text-align: left;
+
+}
+ 
+.portfolio-box-image {
+    overflow: hidden;
+    border-radius: 4px;
+}
+ 
+.portfolio-box-image img {
+    -o-transition: all .8s;
+    -moz-transition: all .8s;
+    -webkit-transition: all .8s;
+    -ms-transition: all .8s;
+    transition: all .8s;
+}
+ 
+.portfolio-box-image:hover img {
+    -o-transform: scale(1.25);
+    -moz-transform: scale(1.25);
+    -webkit-transform: scale(1.25);
+    -ms-transform: scale(1.25);
+    transform: scale(1.25);
+}
+ 
+.portfolio-box h3 {
+    margin-top: 30px;
+}
+.portfolio-box h3 a { color: #555; }
+.portfolio-box h3 a:hover, 
+.portfolio-box h3 a:focus { color: #9b59b6; }
+ 
+.portfolio-box h3 i {
+    padding-left: 10px;
+    color: #ccc;
+    -o-transition: all .3s;
+    -moz-transition: all .3s;
+    -webkit-transition: all .3s;
+    -ms-transition: all .3s;
+    transition: all .3s;
+}
+.portfolio-box h3:hover i { padding-left: 15px; }
+ 
+.portfolio-box-date {
+    padding-bottom: 10px;
+    font-size: 14px;
+    color: #aaa;
+    text-transform: uppercase;
+}
+ 
+.portfolio-box-date i {
+    padding-right: 5px;
+    color: #ddd;
+}
+
+/*photo gallery section*/
+
+.image-container{
+    margin: 0px auto; // center
+  width:1100px; 
+  height:500px; 
+  text-align:center; 
+}
+.carousel-inner{
+ height: 500px;  
+ 
+ }
+
+
+ /*footer*/
+
+.footer_area {
+  margin-top: 100px;
+  background: lightgrey; 
+}
+
+.f_title {
+  margin-bottom: 35px; }
+  .f_title h3 {
+    color: #fff;
+    font-size: 18px;
+    font-weight: bold;
+    font-family: "Heebo", sans-serif;
+    margin-bottom: 0px; }
+
+.ab_widget p {
+  font-size: 14px;
+  line-height: 24px;
+  font-family: "Roboto", sans-serif;
+  color: #777777;
+  margin-bottom: 30px; }
+  .ab_widget p a {
+    color: #766dff; }
+  .ab_widget p + p {
+    margin-bottom: 0px; }
+
+.social_widget p {
+  font-size: 14px;
+  line-height: 24px;
+  font-family: "Roboto", sans-serif;
+  color: #777777;
+  margin-bottom: 10px; 
+}
+.social_widget .list li {
+  margin-right: 10px;
+  display: inline-block; 
+  size: 10px;
+}
+  .social_widget .list li a {
+    font-size: 20px;
+    transition: all 300ms linear 0s; }*/
+  .social_widget .list li:hover a {
+    color: #766dff; }
```

Comment:
consistent indentation, here and elsewhere in the the file? should we put the `}` on newline like other places in the file?